Makefile

```diff
@@ -33,18 +33,22 @@ T_RESET := \e[0m
 .PHONY: all
 all: 1.20 1.21 1.22 1.23 ## Build all versions of EKS Optimized AL2 AMI
 
+# ensure that these flags are equivalent to the rules in the .editorconfig
+SHFMT_FLAGS := --list \
+--language-dialect auto \
+--indent 2 \
+--binary-next-line \
+--case-indent \
+--space-redirects
+
+SHFMT_COMMAND := $(shell which shfmt)
+ifeq (, $(SHFMT_COMMAND))
+SHFMT_COMMAND = docker run --rm -v $(MAKEFILE_DIR):$(MAKEFILE_DIR) mvdan/shfmt
```

Comment:
Is this actually better? Is it a big deal to set this up?

Reply:
I think it's handy because:
1. you can always use the latest `smfmt` without thinking about it
2. if you use multiple machines for dev (like we do) you don't have to do any setup/maintenance on your various boxes.
3. `shfmt` requires a working `go` setup, which isn't a given for our contributors.